test/unit/path_spec.lua

```diff
@@ -293,6 +293,60 @@ describe('path_shorten_fname_if_possible', function()
   end)
 end)
 
+describe('path.c path_guess_exepath', function()
+  local cwd = lfs.currentdir()
+
+  for _,name in ipairs({'./nvim', '.nvim', 'foo/nvim'}) do
+    itp('"'..name..'" returns name catenated with CWD', function()
+      local name = to_cstr(name)
```

Comment:
BTW, most of time you may remove `to_cstr` and just use lua strings. Luajit will handle necessary conversions.